src/classes/services/crawl_manager.py

```
@@ -0,0 +1,306 @@
+        Extract domain from URL
+        
+        Args:
+            url: URL to extract domain from
+            
+        Returns:
+            Domain string
+        """
+        try:
+            parsed = urlparse(url)
+            return parsed.netloc
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)